src/Microsoft.Azure.WebJobs.Host/Dispatch/DispatchQueueHandler.cs

```diff
@@ -0,0 +1,36 @@
+﻿// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the MIT License. See License.txt in the project root for license information.
+
+using System.Threading;
+using System.Threading.Tasks;
+using Newtonsoft.Json.Linq;
+
+namespace Microsoft.Azure.WebJobs.Host.Dispatch
+{
+    /// <summary>
+    /// Queue where user can add their function triggering messages,
+    /// these messages will be distributed to multiple worker instance for processing
+    /// It is a wrapper around <see cref="SharedQueueHandler"/>
+    /// </summary>
+    public class DispatchQueueHandler
```

Comment:
> public class DispatchQueueHandler [](start = 3, length = 34)

Nit: Can the exposes type be an interface? Then this specific class is just an internal impl of that interface. 
1. That's symmetric with ITriggeredFunctionExecutor 
2. It's easier to mock in testing (whereas this class can't be mocked)